app/controllers/api/v1/application_controller.rb

```diff
@@ -5,9 +5,20 @@ module V1
     class ApplicationController < ActionController::API
       include Pundit::Authorization
       include ErrorRenderable
+      include Rescuable
 
       before_action :doorkeeper_authorize!
 
+      def raw_resources(klass, policy_scope_klass = nil)
```

Comment:
```suggestion
      def authorized_resources(klass, policy_scope_klass = nil)
```

The name `raw_resources` was added after we had the `resources` method and mean that it was the resources without pagination.
But if I could travel back in time, I would probably choose another name :harold:.

As this case is simpler (no search, filter, ...) I recommend to give it an explicit name like `authorized_resources` 🙇

Reply:
Also, be careful, it might conflict with https://github.com/nimblehq/ic-rails-phong-bliss/pull/54#discussion_r1243232823